app/views/notifier/notify.html.haml

```diff
@@ -18,6 +18,7 @@
   %br/
   = link_to "View this message in your inbox", notification_url(@notification)
   %br/
-  = link_to "Turn off these notifications", edit_member_registration_url
+  = link_to "Turn off these notifications", unsubscribe_member_url(@signed_message)
```

Comment:
I'm happy to merge this PR but I would like this string changed to "Unsubscribe from these notifications" (with the word "Unsubscribe" linked) as I think that's what people scan for when looking for a link to click.